PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/build.gradle

```diff
@@ -1 +1,10 @@
+dependencies {
+    // as in the azure-toolkit-for-intellij module
+    compile 'com.microsoft.azuretools:azuretools-core:3.53.0-SNAPSHOT', {
```

Comment:
as mentioned in other PRs, azuretools-core is to be discarded, so please don't add dependency on it.